### DISCUSSION TITLE
Extension has improved a ton!

### DISCUSSION THREAD

**Discussion by mattwelke** [golang/vscode-go#1155] [General]
I just tried the extension again today. I used to use it in late 2018 and found it was too unstable. Maybe that had to do with the fact I  was using it with Go Modules, and I remember the Go world was a bit chaotic back then.

I tried it again in summer of 2020 and found it was a lot nicer. And then I tried it again today. It feels *buttery smooth* now, in every way possible. I felt no friction getting started. I had no settings to change. Features like code completion, import suggestions (whether from standard library or my own code), debugger, all work perfectly and were lightning fast.

I'm mega impressed by this. Kudos to the folks who made all this stuff happen!

**Comment by stamblerre** [comment]
Thank you for the kind words, @mattwelke! We really appreciate that you've given the extension multiple tries. We'll continue to work on improving the experience and look forward to hearing more feedback from you in the future 😃
